awxkit/requirements.txt

```diff
@@ -1,8 +1,4 @@
 PyYAML>=5.1
-cryptography
-flake8
 python-dateutil
 requests
 termcolor
```

Comment:
@shanemcd I think `termcolor` is _maybe_ the only thing on this list that might require some work on our end